### PR TITLE
[#125833161] Feature: Introduce IP geocoding liquid filter

### PR DESCRIPTION
**Pull request by Tonkonozhenko**

### CODE REVIEW

File: source/campaigns/editor/filters.rst [hunk 1]
```diff
@@ -509,4 +509,21 @@ Refresh the screenshot every week:
 
    <img src="{{ 'www.example.com' | url2png: ttl: 604800 }}" />
 
+geocode
+.......
+
+Returns location for IP you pass to it.
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst]: "Returns location for a given IP"

**Comment by zhuravel** [comment]
Looks ok.
Should we add `ip_address` to origin's liquid payload?
Because right now you can't get the origin's IP which makes this filter kinda useless.

**Comment by Tonkonozhenko** [comment]
is [this](https://github.com/talkable/talkable/pull/1601/files#diff-e7bca0e75b42c18c6e6e3b5d08958c0eL11) incorrect?

**Comment by zhuravel** [comment]
I'm not entirely sure what was the reason behind adding this filter, but if we want to use it in Campaign Join criteria, we need to add `ip_address` to `Origin#to_payload`.

**Pull request closed without merging**